Tests/PubNubTests/Networking/Routers/ChannelGroupEndpointTests.swift

```diff
@@ -284,36 +282,36 @@ final class ChannelGroupsEndpointTests: XCTestCase {
         case .success:
           XCTFail("Add Channel request should fail")
         case let .failure(error):
-          XCTAssertEqual(error.pubNubError, PubNubError(reason: .invalidCharacter))
+          XCTAssertEqual(error.pubNubError, PubNubError(.invalidCharacter))
         }
         expectation.fulfill()
       }
 
     wait(for: [expectation], timeout: 1.0)
   }
+}
 
-  // MARK: - Group Channel Remove
+// MARK: - Remove Channel from Group
 
-  func testGroupChannelsRemove_Endpoint() {
-    let endpoint = Endpoint.removeChannelsForGroup(group: testGroupName, channels: testChannels)
+extension ChannelGroupsRouterTests {
+  func test_RemoveChannelsForGroup_Router() {
+    let router = ChannelGroupsRouter(.removeChannelsForGroup(group: testGroupName, channels: testChannels), configuration: config)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line should be 120 characters or less: currently 130 characters](https://app.codacy.com/app/PubNub/swift/pullRequest?prid=4513555)